Makefile

```diff
@@ -18,14 +15,15 @@ help: ## This help.
 
 .PHONY: docker
 docker: ## Build Docker image
-	@(echo "Building docker image...")
-	docker build --file ./Dockerfile --build-arg VERSION=${VERSION} \
-		--build-arg BUILD_TIMESTAMP=$(shell date -u +"%Y-%m-%dT%H:%M:%SZ") \
-		--build-arg COMMIT_HASH=$(shell git rev-parse HEAD) \
-		-t ${DOCKER_IMAGE}:${DOCKER_TAG} .
-
+	$(info "Building docker image...")
+	# build both images
+	docker buildx build --platform=linux/amd64,linux/arm64 -t $(DOCKER_IMAGE):$(DOCKER_TAG) .
+	# load just the current platform
+	# https://github.com/docker/buildx/issues/59#issuecomment-1168619521
```

Comment:
Why we can't just build & load it just for the current platform (`docker buildx build --load --platform linux/$(shell uname -m) -t $(DOCKER_IMAGE):$(DOCKER_TAG) .`?
Why we need to build for both if we load only for the current platform?

Reply:
If this is just for test mutil-arch maybe we need a different make command for it?
They way it is written here, the image is being build 3 times - not sure why we need it

Reply:
I have spoken to @chrisgacsal and we agreed that for now it's fine as it is and later if we need to test multi arch builds locally then we decide how to move forward.